src/libraries/System.Linq.Expressions/ref/System.Linq.Expressions.cs

```diff
@@ -1205,4 +1228,17 @@ public partial class RuleCache<T> where T : class
     {
         internal RuleCache() { }
     }
+    public static partial class RuntimeOps
+    {
+        public static System.Runtime.CompilerServices.IRuntimeVariables CreateRuntimeVariables() { throw null; }
+        public static System.Runtime.CompilerServices.IRuntimeVariables CreateRuntimeVariables(object[] data, long[] indexes) { throw null; }
+        public static bool ExpandoCheckVersion(System.Dynamic.ExpandoObject expando, object? version) { throw null; }
+        public static void ExpandoPromoteClass(System.Dynamic.ExpandoObject expando, object oldClass, object newClass) { }
+        public static bool ExpandoTryDeleteValue(System.Dynamic.ExpandoObject expando, object? indexClass, int index, string name, bool ignoreCase) { throw null; }
+        public static bool ExpandoTryGetValue(System.Dynamic.ExpandoObject expando, object? indexClass, int index, string name, bool ignoreCase, out object? value) { throw null; }
+        public static object? ExpandoTrySetValue(System.Dynamic.ExpandoObject expando, object? indexClass, int index, object? value, string name, bool ignoreCase) { throw null; }
+        public static System.Runtime.CompilerServices.IRuntimeVariables MergeRuntimeVariables(System.Runtime.CompilerServices.IRuntimeVariables first, System.Runtime.CompilerServices.IRuntimeVariables second, int[] indexes) { throw null; }
+        [return: System.Diagnostics.CodeAnalysis.NotNullIfNotNullAttribute("expression")]
+        public static System.Linq.Expressions.Expression? Quote(System.Linq.Expressions.Expression? expression, object hoistedLocals, object[] locals) { throw null; }
+    }
```

Comment:
There's a ton of change being made to System.Linq.Expressions.cs here, effectively adding / changing public surface area.  Seems like that really should be in its own PR, with the appropriate discussion around it.  Unless I'm missing something?

Reply:
The GenerateReferenceSource tool was adding these classes from source, also it was removing/changing some accessors/modifiers. For me as I don't know if they should be added or not I reverted all change except nullable related ones. Seems most of changes here related to that

Reply:
As @buyaa-n points out, the changes stem from regenerating sources after making [a single change](https://github.com/dotnet/runtime/pull/979/files/dc6b2e31b84d1ee360b5c72ce6897889e7b85505#diff-93045c934a6feac07e6f303bc1596c12R78) in the System.Linq.Expressions implementation. This shouldn't happen, the expectation is that regenerating reference sources without implementations should not produce a diff. This can be controlled by blacklisting members, c.f. https://github.com/dotnet/runtime/blob/master/src/libraries/System.Reflection.Metadata/ref/ReferenceSourceExcludeApi.txt. I think we should probably revisit in a separate PR.

Reply:
Thanks. Yes, please revert all changes not relevant to nullable annotations and open an issue about the differences.

Reply:
Done

Reply:
Thanks.